testing/camunda-process-test-java/src/test/java/io/camunda/process/test/impl/extensions/CamundaProcessTestContextIT.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.process.test.impl.extensions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.client.CamundaClient;
+import io.camunda.client.api.response.DeploymentEvent;
+import io.camunda.client.api.response.ProcessInstanceEvent;
+import io.camunda.process.test.api.CamundaAssert;
+import io.camunda.process.test.api.CamundaProcessTest;
+import io.camunda.process.test.api.CamundaProcessTestContext;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.model.bpmn.BpmnModelInstance;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Future;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+
+@CamundaProcessTest
+public class CamundaProcessTestContextIT {
+  private static final int TIMEOUT = 40;
+  private CamundaProcessTestContext processTestContext;
+  private CamundaClient client;
+
+  @Test
+  void shouldThrowBpmnErrorWithoutVariables() {
+    // Given
+    processTestContext.mockJobWorker("test").thenThrowBpmnError("bpmn-error");
+    final long processDefinitionKey = deployProcessModel();
```

Comment:
:wrench: We could extract the deployment using a `@BeforeEach` method.

Reply:
For this PR, this is true, but in next one we deploy a different models. So we'll continue with just calling deploy manually.